packaging/conda/meta.yaml

```diff
@@ -40,7 +42,7 @@ outputs:
 
   # khiops-core package
   - name: khiops-core
-    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    version: {{ version.replace("-", ".") }}
```

Comment:
I would substitute "-" for "" to be consistent with current versioning scheme (see comment above).